rest-api/test/unit_test/api_test/awardee_api_test.py

```diff
@@ -69,10 +82,11 @@ def test_get_awardee_no_organizations(self):
 
   def test_get_awardee_with_organizations(self):
     self._setup_data()
-    result = self.send_get('Awardee/PITT')
-    self.assertEquals(self._make_expected_pitt_awardee_resource(), result)
+    result = self.send_get('Awardee/PITT?_inactive=true')
```

Comment:
We also need a test for Awardee/PITT not passing in _inactive=true; that should equal _make_expected_pitt_awardee_resource(inactive=False)

Reply:
done